microsoft-edge/extensions-chromium/developer-guide/minimize-page-load-time-impact.md

```diff
@@ -0,0 +1,260 @@
+
+1. To start recording a performance profile, click the **Record** (![Record icon](./minimize-page-load-time-impact-images/record-icon.png)) button.
+
+1. Reload the page to capture the profiling data that corresponds to the page load time and then, once the page has finished loading, click the **Stop** (![Stop icon](./minimize-page-load-time-impact-images/stop-icon.png)) button to end the recording. DevTools displays the recorded performance profile:
+
+   ![The recorded performance profile displayed in the DevTools Performance tool](./minimize-page-load-time-impact-images/profile.png)
+
+1. To search for the performance events that are caused by your content script, press **Ctrl+F** on Windows/Linux or **Command+F** on macOS.  The **Find** text box appears at the bottom of the **Performance** tool.
+
+1. Type **Evaluate script** and then press **Enter** until the **Performance** tool highlights the performance events that are caused by your content script.  You will know that you have found the right performance event when the **Script** label in the **Summary** panel shows the name of your content script.
+
+   ![The Evaluate script performance event due to the extension's content script running during the page load](./minimize-page-load-time-impact-images/evaluate-script-event.png)
+
+
+<!-- ------------------------------ -->
+#### Profile your content script using the Edge tracing tool
+
+The Edge tracing tool, available at the `edge://tracing` URL, is a powerful tool that can provide detailed analysis of your extension's performance. In this section, you'll learn how to use the Edge tracing tool to understand the impact of your extension on the page load time. To learn more about this tracing tool, which is based on the **Perfetto** tool, see [Perfetto UI](https://perfetto.dev/docs/visualization/perfetto-ui) in the Perfetto Tracing Docs.
+
+1. To open the Edge tracing tool, open a new tab or window, and go to `edge://tracing`. The tracing UI opens.
+
+1. To start a new trace, in the top left corner of the tool, click the **Record** button. The **Record a new trace** dialog opens.
+
+1. Select the **Manually select settings** option button. The list of categories appears.
+
+1. To capture detailed information about your extension's content script compilation and execution, select all of the following categories:
+
+   * **extensions**
+   * **v8**
+   * **devtools**
+   * **devtools.timeline**
+
+1. Click the **Record** button.  The dialog closes, and the Edge tracing tool starts recording the trace.
+
+1. Open a new tab and load a webpage that your extension affects. The tracing tool collects data about the performance impact of your extension on the webpage.
+
+1. Open the tab where the Edge tracing tool is running, and then click the **Stop** button. The new trace information appears in the tool.
+
+Traces that are recorded by the Edge tracing tool provide a lot of information about the browser, as well as your extension.
```

Comment:
add heading here
```suggestion


<!-- ------------------------------ -->
#### Filter the results

Traces that are recorded by the Edge tracing tool provide a lot of information about the browser, as well as your extension.
```

Reply:
but the little section at end might need heading or moving it
"To continue analyzing the performance ..."